cli/src/cmd/forge/inspect.rs

```diff
@@ -64,8 +68,8 @@ impl Cmd for InspectArgs {
         // Build the project
         let project = modified_build_args.project()?;
         let outcome = if let Some(ref mut contract_path) = contract.path {
-            let target_path = dunce::canonicalize(&*contract_path)?;
-            *contract_path = target_path.to_string_lossy().into_owned();
+            let target_path = canonicalize(&*contract_path)?;
```

Comment:
Do we still need to canonicalize here?

Reply:
yes because all paths are still absolute

Reply:
but path slashing is now baked into `canonicolize` on windows